### PR TITLE
Ctl parser

### PR DESCRIPTION
**Pull request by gcsantos-gpa**
SC-201
SC-202

If we need to make this value list required I can do that, but I'm not sure it's necessary.

### CODE REVIEW

File: Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/NewMeterWizard/ChannelPage.tsx [hunk 1]
```diff
@@ -342,16 +360,8 @@ export default function ChannelPage(props: IProps) {
         const regex = new RegExp('\(A[0-9]+\)Analog Channel [0-9]+');
 
         const digital = regex.test(ch.Description) && ch.MeasurementType == 'Digital';
-        const sparePhrase = ch.Description != null && [
```
Comment (clackner-gpa) [inline Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/NewMeterWizard/ChannelPage.tsx:345]: we should make `SpareChannel` a requried ValueList if we do this otherwhise we won't have it and it'll be a pain to figure out where it is

File: Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/NewMeterWizard/ChannelPage.tsx [hunk 2]
```diff
@@ -418,9 +428,8 @@ export default function ChannelPage(props: IProps) {
                                 BtnClass={'btn-info' }
```
Comment (clackner-gpa) [inline Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/NewMeterWizard/ChannelPage.tsx:428]: we probably should track loading state of the SpareList request and add a spinning wheel while that is loading too

**Review by clackner-gpa — CHANGES_REQUESTED** [review]

File: Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/NewMeterWizard/ChannelPage.tsx (hunk 2 above)
Comment (gcsantos-gpa) [reply Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/NewMeterWizard/ChannelPage.tsx:428]: I can disable it while loading, but changing the button to a spinning wheel would require a gemstone change. Alternatively, I could add a loading screen to the whole page.

Comment (clackner-gpa) [reply Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/NewMeterWizard/ChannelPage.tsx]: disable is fine

**Review by clackner-gpa — APPROVED** [review]

**Pull request merged**